### PR TITLE
Support for EVAL command for Dynomite

### PR DESCRIPTION
**Pull request by shailesh33**

### CODE REVIEW

File: dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java [hunk 1]
```diff
@@ -429,6 +431,62 @@ public Long execute(Jedis client, ConnectionContext state) {
         });
     }
 
+    @Override
+    public Object eval(String script, int keyCount, String... params) { return d_eval(script, keyCount, params).getResult(); }
+
+    public OperationResult<Object> d_eval(final String script, final int keyCount, final String... params) {
+        if (keyCount == 0) {
+            throw new DynoException("Need atleast one key in script");
```
Comment (ipapapa) [inline dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: `at least`

File: dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java [hunk 2]
```diff
@@ -429,6 +431,62 @@ public Long execute(Jedis client, ConnectionContext state) {
...
+        throw new UnsupportedOperationException("This function is Not Implemented. Please use eval instead.");
+    }
+
+    @Override
+    public Object evalsha(String script) {
+        throw new UnsupportedOperationException("This function is Not Implemented. Please use eval instead.");
+    }
+
+    @Override
+    public Boolean scriptExists(String sha1) {
```
Comment (ipapapa) [inline dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: What is the issue and we cannot implement `scriptExists`?

Comment (shailesh33) [reply dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: which node will you ask if a script exists? there is no key in a script till it is executed

File: dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java (hunk 1 above)
Comment (shailesh33) [reply dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: Done

**Pull request merged**